addons/website/models/website.py

```diff
@@ -852,6 +852,11 @@ def fallback_create_missing_industry_image(image_name, fallback_img_name):
             fallback_create_missing_industry_image('s_text_cover_default_image', 's_cover_default_image')
             fallback_create_missing_industry_image('s_showcase_default_image', 's_image_text_default_image')
             fallback_create_missing_industry_image('s_accordion_image_default_image', 's_image_text_default_image')
+            fallback_create_missing_industry_image('s_image_title_default_image', 's_cover_default_image')
```

Comment:
Why you add this line ? It seems related to the `s_image_title` snippet but I don't know/find this snippet

Reply:
Oops, I forgot to remove this leftover, thanks !